src/app/shared/mastodon/mastodon.component.ts

```diff
@@ -0,0 +1,152 @@
+/**
+ * Mastodon embed feed timeline v3.8.2
+ * More info at:
+ * https://gitlab.com/idotj/mastodon-embed-feed-timeline
+ */
+
+import { Component } from '@angular/core';
+import { MastodonService } from './mastodon.service'; // Import the service
+
+export interface MastodonTimelineData {
+  postUrl: string;
+  accountUrl: string;
+  accountAvatar: string;
+  reblogAccountAvatar: string;
+  accountUsername: string;
+  postDate: string;
+  postContent: string;
+  spoilerText: string;
+  mediaContent: any[];
+  previewLink: string;
+  poll: any[];
+}
+
+@Component({
+  selector: 'app-mastodon-timeline',
+  templateUrl: './mastodon.component.html',
+  styleUrls: ['./mastodon.component.scss'],
+})
+export class MastodonComponent {
+  fetchedData: Map<string, (string | number | boolean)[]>;
+  timelineData: MastodonTimelineData[] = [];
+  containerBodyId: string = 'mt-body'; // Id of the <div> containing the timeline
+  defaultTheme: string = 'light'; // Preferred color theme: 'light', 'dark' or 'auto'. Default: auto
+  instanceUrl: string = 'https://genomic.social'; // Your Mastodon instance
+  timelineType: string = 'profile'; // Choose type of toots to show in the timeline: 'local', 'profile', 'hashtag'. Default: local
+  userId: string = '110973634882132620'; // Your user ID on Mastodon instance. Leave empty if you didn't choose 'profile' as type of timeline
+  profileName: string = 'dockstore'; // Your user name on Mastodon instance. Leave empty if you didn't choose 'profile' as type of timeline
+  hashtagName: string = ''; // The name of the hashtag. Leave empty if you didn't choose 'hashtag' as type of timeline
+  tootsLimit: string = '2'; // Maximum amount of toots to get. Default: 20
```

Comment:
I feel like we could show more than 2 posts and add a scroll bar to the mastodon component if the content overflows.

Reply:
Now shows a max of 5